firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/AabUpdater.java

```diff
@@ -48,18 +50,22 @@
   private AppDistributionReleaseInternal aabReleaseInProgress;
 
   private final Object updateAabLock = new Object();
+  private final Context applicationContext;
```

Comment:
Nit: for brevity, can we just call this `context` everywhere instead of `applicationContext`?

Reply:
I did see we used context elsewhere. Context being so vague is the reason I added application to it, but I have no issue making it context.